deploy-agent/deployd/common/env_status.py

```diff
@@ -42,13 +42,13 @@ def load_envs(self):
             with self._lock, open(self._status_fn, 'r+') as config_fn:
                 data = json.load(config_fn)
                 log.debug('load status file: {}'.format(data))
-                envs = {key: DeployStatus(json_value=d) for key, d in data.iteritems()}
+                envs = {key: DeployStatus(json_value=d) for key, d in data.items()}
         except IOError:
             log.info("Could not find file {}. It happens when run deploy-agent the "
                      "first time, or there is no deploy yet.".format(self._status_fn))
             return {}
-        except Exception as e:
-            log.error(e.message)
+        except Exception:
+            log.exception("Something went wrong in load_envs")
```

Comment:
would be nice to log the exception messsage as the original intent

Reply:
log.exception already does that